client/blocks/credit-card-form/helpers.js

```diff
@@ -139,8 +139,14 @@ export function getParamsForApi( cardDetails, cardToken, stripeConfiguration, ex
 
 export function useDebounce( value, delay ) {
 	const [ debouncedValue, setDebouncedValue ] = useState( value );
-	const debounced = useRef( debounce( newValue => setDebouncedValue( newValue ), delay ) );
```

Comment:
It wasn't obvious to me why this needs to move away from the `lodash` version, but I'll trust there's a reason :)

Reply:
It actually probably doesn't. I was trying to fix the tests and I thought that maybe the lodash debounce was causing problems, but it turns out it's just that enzyme shallow mounting will not run `useEffect` hooks yet. I could probably remove this change.

Reply:
Done.